cds/modules/deposit/indexer.py

```diff
@@ -46,6 +46,11 @@ def cdsdeposit_indexer_receiver(
     if record['$schema'] in [project_schema, video_schema]:
         json['_cds']['state'] = deposit['_cds']['state']
         json['_files'] = deposit['_files']
+        if json.get('_access'):
```

Comment:
We need this in order to be able to search by `_access.[read,update]`. The record/deposit will hold the original value while the indexed record will force to lowercase. This is needed [here](https://github.com/CERNDocumentServer/cds-videos/blob/main/cds/modules/deposit/search.py#L75). An alternative would require some ES mapping update.